src/components/Avatar/Avatar.vue

```diff
@@ -100,6 +100,14 @@ import Tooltip from '../../directives/Tooltip'
 import usernameToColor from '../../functions/usernameToColor'
 import { userStatus } from '../../mixins'
 
+function getUserHasAvatar(userId) {
+	return window.sessionStorage.getItem('userHasAvatar-' + userId)
```

Comment:
We could probably apply a similar naming schema as we discusssed for the event names in https://github.com/nextcloud/nextcloud-event-bus/issues/131#issuecomment-683689107 That way we still can figure out where a value might be coming from when we fill out browser/session storage more over time.

@skjnldsv @ChristophWurst Didn't we have some planned prefix for the components library as well?

Basically someting like `vue-components:user-has-avatar.userID`

Reply:
Nah, components were generic, so `nextcloud:user-has-avatar.userID` iirc :thinking:

Reply:
https://github.com/nextcloud/nextcloud-browser-storage :thinking:

Reply:
ah, that's the global thing I was looking for, thanks :-)

I'll adjust